RELEASE.md

```diff
@@ -1,34 +1,64 @@
 # Release Process
 
-## Requirements
+This documentation guides you on how to release a new version of the Codacy Chart.
 
-Please make sure you have these tools installed before starting this process:
+The release process consists of the following stages:
 
--   git
--   [ytool](https://github.com/codacy/ytool)
--   [helm >=3.x](https://helm.sh/docs/intro/install/)
+1.  **Deciding to release a new version**
 
-## Prepare a new release
+    Decide that we're ready to release a new version of the Codacy Chart and elect who will be the Release Manager.
 
--   [ ] 1.  Inform the engineering team that you are the release manager for a new release in #team-engineering in our slack (tag engineers) and get a status from each squad on any changes that may be on the master branch of components that may be dealbreakers for the release.
+2.  **Preparing the release for testing**
 
--   [ ] 2.  Clone this project on master branch
+    The Release Manager creates a release candidate branch that will be used to test the new release.
+
+3.  **Testing and stabilizing the release**
+
+    The Release Manager involves the QA team and relevant stakeholders in testing the new release, and applies fixes for bugs that are detected.
+
+4.  **Launching the new release**
```

Comment:
Codacy found an issue: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=5647605)